new-library.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
```

Comment:
We want to get rid of ruby dependency, it was a mistake :p Would be nice to convert it in js while it's small and easy